docs/src/locators.md

```diff
@@ -629,7 +590,56 @@ page.GetByRole("section"), new() { Has = page.GetByTestId("subscribe-button") })
 
 Note that inner locator is matched starting from the outer one, not from the document root.
 
-#### Augment an existing locator
+### Locate in a subtree
```

Comment:
This is not really a part of "Locate elements that contain other elements". Perhaps we can do more sections?